ui/src/tool-panel/location-track/location-track-location-infobox.tsx

```diff
@@ -223,6 +227,28 @@ export const LocationTrackLocationInfobox: React.FC<LocationTrackLocationInfobox
                             },
                         };
                     });
+                    const startSwitchId =
+                        extraInfo.startSplitPoint.type === 'SWITCH_SPLIT_POINT'
+                            ? extraInfo.startSplitPoint.switchId
+                            : undefined;
+                    const endSwitchId =
+                        extraInfo.endSplitPoint.type === 'SWITCH_SPLIT_POINT'
+                            ? extraInfo.endSplitPoint.switchId
+                            : undefined;
+
+                    const prefilledSplits = prefilled
```

Comment:
Prefillattujen splitpointtien haku tehdään moniosaisesti. Ensimmäinen osa, jossa käydään alustavasti läpi potentiaaliset katkaisukohdat (eli duplikaattiraiteiden alku- ja loppupisteet,) tehdään täällä. Loppu tehdään `split-store.ts`:ssä. Jako on tehty toistaiseksi puhtaasti datamallin perusteella - täällä käsitellään ja lähetetään eteenpäin `SplitPoint`-tyyppiä ja storessa tehdään tarkemmat tarkastelut kullekin pisteelle ja muunnokset lopulliseen `SplitTargetCandidate`-tyyppiin